src/Disks/ObjectStorages/AzureBlobStorage/AzureObjectStorage.h

```diff
@@ -40,7 +40,7 @@ class AzureObjectStorage : public IObjectStorage
 
     ObjectStorageType getType() const override { return ObjectStorageType::Azure; }
 
-    std::string getCommonKeyPrefix() const override { return ""; }
```

Comment:
I wonder why it was not defined in the first place, and why there is no single comment about that.